src/backend/app/core/config.py

```diff
@@ -39,51 +48,57 @@ def sentry_dsn_can_be_blank(cls, v: str) -> str | None:
     POSTGRES_DB: str
     SQLALCHEMY_DATABASE_URI: PostgresDsn | None = None
 
-    @validator("SQLALCHEMY_DATABASE_URI", pre=True)
-    def assemble_db_connection(cls, v: str | None, values: dict[str, Any]) -> Any:
+    @field_validator("SQLALCHEMY_DATABASE_URI", mode="before")
+    def assemble_db_connection(cls, v: str | None, info: ValidationInfo) -> Any:
+        """
+        A function that validates and assembles the database connection string.
+        Args:
+            v (str | None): The database URI string.
+            info (ValidationInfo): Pydnatic Information related to the validation process.
+        Returns:
+            Any: The assembled database connection string.
+        """
```

Comment:
```suggestion
```

We can remove docstrings for now to keep everything consistent.

Reply:
done!